guides/Other Subjects/CreatingPointsSystems.md

```diff
@@ -0,0 +1,161 @@
+Creating a points system (also known as **Social Module**) in your bot is quite easy in Klasa, as we will use our configuration system: **SettingGateway**. You do NOT need to rewrite any of your code to change the provider, but we also suggest to **NOT** use the JSON provider for this in production, as you may reach the limit of files you can open simultaneously in your [OS](https://en.wikipedia.org/wiki/Operating_system).
+
+## Setting the schema
+
+Before we work with the social module, we need to update the built-in {@link GatewayDriver#users users' gateway} to implement a new key:
+
+```javascript
+async function init() {
+	if (!this.client.gateways.users.schema.has('experience')) {
+		this.client.gateways.users.schema.add('experience', {
+			type: 'integer',
+			default: 0,
+			configurable: false
+		});
+	}
+}
+```
+
+Where we'd check if the schema has the key `experience`. If it doesn't, we add it as a new key, with type `integer` (doubtfully we'll use `float` on this) and make it unconfigurable for the built-in userconf command so the end users do not cheat by modifying their stats.
```

Comment:
"Where we'd check if the schema..." should be changed to "this is where we'd check if the schema..."